apps/nextjs-website/src/components/HowTo.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+import { HowToProps } from '@pagopa/pagopa-editorial-components/dist/components/HowTo';
+import { HowTo as HowToEC } from '@pagopa/pagopa-editorial-components';
+import { Icon } from '@mui/material';
+import { formatValidMuiIcon, isValidMuiIcon } from './Icons';
+import { HowToSection } from '@/lib/fetch/types/PageSection';
+
+const makeHowToProps = ({
+  rowMaxSteps,
+  link,
+  steps,
+  ...rest
+}: HowToSection): HowToProps => ({
+  ...(rowMaxSteps && { rowMaxSteps }),
+  ...(link && {
+    link: {
+      href: link.href,
+      label: link.text ?? link.href,
+    },
+  }),
+  steps: steps.map((step) => ({
+    title: step.title,
+    description: step.description, // TODO: Parse rich text (markdown)
```

Comment:
I would try using the [component for rendering Markdown](https://github.com/pagopa/b2b-portals/blob/093765e/apps/nextjs-website/src/components/MarkdownRenderer.tsx#L10)